internal/cmd/version.go

```diff
@@ -2,12 +2,13 @@ package cmd
 
 import (
 	"fmt"
+
 	"github.com/spf13/cobra"
 )
 
 func versionCmd(version string) *cobra.Command {
 	return &cobra.Command{
-		Use: "version",
+		Use:   "version",
```

Comment:
good, maybe we can add a `format` command to the makefile to run `go-fmt`. what do you think?